guides/micronaut-cloud-trace-google/metadata.json

```
@@ -0,0 +1,13 @@
+{
+  "base": "micronaut-cloud-trace-base",
+  "title": "OpenTelemetry Tracing with Google Cloud Trace and the Micronaut Framework",
+  "intro": "Use Google Cloud Trace to investigate the behaviour of your Micronaut applications.",
```

```suggestion
  "intro": "Use Google Cloud Trace to investigate the behavior of your Micronaut applications.",
```
Change to American spelling optional